ios/engine/KMEI/KeymanEngine/Info.plist

```diff
@@ -2,6 +2,8 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>KeymanVersionWithTag</key>
+	<string>10.0.0-local</string>
```

Comment:
Is this just a dummy version that is ignored?

Reply:
For the item to exist so that our version-update script can insert it, we do have to define _something_ here.  I chose to go with something obviously wrong so that the default value will be a "canary" of sorts - we'll know quickly if something breaks in the versioning scripts if we ever see it.